jekyll/_cci2/private-images.md

```diff
@@ -19,18 +19,19 @@ jobs:
           password: $DOCKERHUB_PASSWORD  # or project environment variable reference
 ```
 
-You can also use images from a private repository like [gcr.io](https://cloud.google.com/container-registry) or [quay.io](https://quay.io)—make sure to supply the full registry/image URL for the `image` key, and use the appropriate username/password for the `auth` key. For example:
+You can also use images from a private repository like [gcr.io](https://cloud.google.com/container-registry) or [quay.io](https://quay.io). Make sure to supply the full registry/image URL for the `image` key, and use the appropriate username/password for the `auth` key. For example:
 
-```
+```yaml
```

Comment:
I think this code block is the cause of the failed build, it's missing the `jobs:` `job-name:` and `docker` keys.